hebiros/src/hebiros_group.cpp

```diff
@@ -0,0 +1,10 @@
+#include "hebiros_group.h"
+
+HebirosGroup::HebirosGroup() {
```

Comment:
Why not just declare this as a `default` impl in the class definition?

Reply:
Primarily because I plan on changing this soon -- there will be more constructor arguments, and so this will be a smaller change in the future.